src/dialogs/generic/show-dialog-box.ts

```diff
@@ -0,0 +1,28 @@
+import { fireEvent } from "../../common/dom/fire_event";
+
+export interface DialogBoxParams {
+  confirmation?: boolean;
+  prompt?: boolean;
+  confirmText?: string;
+  dismissText?: string;
+  inputLabel?: string;
+  inputType?: string;
+  text?: string;
+  title?: string;
+  cancel?: () => void;
+  confirm?: (out?: string) => void;
+}
+
+export const loadGenericDialog = () =>
+  import(/* webpackChunkName: "confirmation" */ "./dialog-box");
+
+export const showDialog = (
```

Comment:
Create 3 helper functions, for each type of dialog so, `showConfirmationDialog` that sets `confirmation` to true, `showAlertDialog` and `showPromptDialog`.

All with their own params type, so I can not set an input type on a confirm dialog, etc.